sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -59,15 +59,14 @@ class ConfigurationSetting(Model):
     }
 
     kind = "Generic"
-    content_type = None
```

Comment:
It's technically a breaking change if this model was public

Reply:
Added back